src/Numerics/Mesh/Grids.jl

```diff
@@ -301,6 +301,8 @@ struct DiscontinuousSpectralElementGrid{
         (vgeo, sgeo, x_vtk) =
             computegeometry(topology.elemtocoord, D, ξ, ω, meshwarp)
 
+        vgeo = vgeo.array
+        sgeo = sgeo.array
```

Comment:
@simonbyrne and @jkozdon  I put here these assignments so that tests that use the driver configs could pass. But now I still have other tests (see CI failure) that do not pass. I don't know what's the best way to fix these issues without having to change everywhere in the code where `vgeo` and `sgeo` are used assuming the old array type. Any help would be much appreciated, thanks.

Reply:
Thanks, Jeremy! He helped me out identify where the issue was

Reply:
No problem. Glad I could be a second second set of eyes for you.